### DISCUSSION TITLE
Send Otp Using Otp Template

### DISCUSSION THREAD

**Discussion by academicmantraSahil** [netflie/laravel-notification-whatsapp#38] [Q&A] [answered]

**Comment by RVP04** [comment]
what is the exact problem

**Comment by sahilsharma100** [comment]
@RVP04 is their any way to push otp using default template provided by whatsapp now

**Comment by RVP04** [comment]
> @RVP04 is their any way to push otp using default template provided by whatsapp now

Sorry sir I have not yet tested the package. I am waiting for my reply

**Comment by academicmantraSahil** [comment, marked as answer]
@RVP04 Solved

 return WhatsAppTemplate::create()
            ->name('otp_verification') // Name of your configured template
            ->language('en')
            ->body(Component::text("123456"))
            ->buttons(Component::urlButton(['123456']));
